.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
The comments should be preceded by a '#' symbol to indicate that they are comments.